app/package.json

```diff
@@ -38,7 +38,8 @@
     "@rdfjs/namespace": "^2.0.1",
     "@reach/auto-id": "^0.15.3",
     "@sentry/nextjs": "^7.112.2",
-    "@storybook/nextjs": "^8.0.9",
+    "@storybook/nextjs": "^8.1.0",
+    "@storybook/react-vite": "^8.1.0",
```

Comment:
Should we have two Storybook frameworks?

Reply:
No, we should not, thanks for noticing. This is a test I made, because I've noticed that the fast refresh was not working in Storybook (we have fast refresh in the app, not in storybook :/). I've removed now the @storybook/react-vite package.